cl/settings/third_party/elasticsearch.py

```diff
@@ -231,3 +231,20 @@
 ELASTICSEARCH_PARALLEL_BULK_THREADS = env(
     "ELASTICSEARCH_PARALLEL_BULK_THREADS", default=5
 )
+
+
+##########################
+# Sweep indexer settings #
+##########################
+ELASTICSEARCH_SWEEP_INDEXER_ACTION = env(
+    "ELASTICSEARCH_SWEEP_INDEXER_ACTION", default="all"
+)
+ELASTICSEARCH_SWEEP_INDEXER_QUEUE = env(
+    "ELASTICSEARCH_SWEEP_INDEXER_QUEUE", default="celery"
+)
```

Comment:
I think we should make a special task queue for this. I'll deploy one called `es_sweep`. Can you set that as the default?

Reply:
Done, I've set `es_sweep` as the default queue!